### DISCUSSION TITLE
each time you horizontally drag the code section, it scrolls to its very top

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3626] [Mobile] [answered]
as titled

https://user-images.githubusercontent.com/39104740/112762095-b3762500-9006-11eb-972b-495c20501388.mp4

**Comment by aconsuegra** [comment, marked as answer]
Thanks for the feedback! This looks like a bug on our side, and it would be helpful for us to know more details about the device and Android OS version you are using.

**Reply by nt4f04uNd** [reply]
xiaomi redmi note 5 android 9
